### ISSUE TITLE
Installation via brew installs 4.0.2 but version shows 4.0.0

### ISSUE THREAD

**Issue by Kondamon** [Flinesoft/BartyCrouch#145]
Hello!

I have installed bartycrouch via homebrew:

==> Downloading https://homebrew.bintray.com/bottles/bartycrouch-4.0.2.mojave.bottle.tar.gz
Already downloaded: /Users/chris/Library/Caches/Homebrew/downloads/d9f3481a2963be759e1392565aaaf00a69c5bf4f9a0b956c30d191606868f263--bartycrouch-4.0.2.mojave.bottle.tar.gz
==> Pouring bartycrouch-4.0.2.mojave.bottle.tar.gz
🍺  /usr/local/Cellar/bartycrouch/4.0.2: 6 files, 6.0MB

However, after checking the version, I receive an old version "4.0.0". How can I resolve this?

    bartycrouch version
    Version: 4.0.0

I have already removed bartycrouch via homebrew and installed it again but with the same result.

**Update:** I have now manually build bartycrouch via make and now I receive version 4.0.2 after running 'bartycrouch version'. So there seems to be some trouble in homeBrew. Can someone reproduce my error?

**Comment by Dschee** [comment]
This is due to the fact that bumping the CLI file was forgotten in releases 4.0.1 and 4.0.2. But this issue was already fixed in a1033ef250820587186ce63ee7ab183cddb15162 and should not appear anymore when we do an update to version 4.0.3 or higher.

**Comment by slashlos** [comment]
I'm trying to first fetch localized assets, but it failing due to this - installed via brew a few days ago
```
bartycrouch version
Version: 4.3.1
```
with this code
```
if let asset = NSDataAsset.init(name: NSLocalizedString(fromAsset, comment: "")) {/* FAILS HERE */
	text = String.init(data: asset.data, encoding: .utf8)
}
else
if let asset = NSDataAsset.init(name: fromAsset) {
	text = String.init(data: asset.data, encoding: .utf8)
}
else
{
	return String(format: "Unable to locate asset:\n%@", fromAsset)
}
````
